packages/flutter_tools/test/general.shard/asset_bundle_test.dart

```diff
@@ -24,18 +24,16 @@ import 'package:standard_message_codec/standard_message_codec.dart';
 import '../src/common.dart';
 import '../src/context.dart';
 
-const String shaderLibDir = '/./shader_lib';
-
 void main() {
-  group('AssetBundle.build', () {
-    late Logger logger;
+  const String shaderLibDir = '/./shader_lib';
+
+  group('AssetBundle.build (non-hermetic)', () {
```

Comment:
in what ways are these non-hermetic? ideally we would either make them hermetic, or move them to the integration shard if they must use the real environment.

Reply:
I'm uncomfortable calling anything that uses `testUsingContext` hermetic. If you ignore the lack of a `Logger` override, these tests are _probably_ hermetic, so perhaps the `non-hermetic` label is unfair. If you have better ideas for group names, I'm all ears (perhaps "using context" and "not using context").

The reason the groups are separate is that one uses `setUp` (polluting the scope with `testFileSystem` and `platform`) and `testUsingContext` (both of which are not recommended for use), and the latter group is more idiomatic. 

I do not wish the rewrite all of these tests to be idiomatic now, as this would be a significant undertaking, and I lack the needed bandwidth.

Reply:
addressed in [tweak group name](https://github.com/flutter/flutter/pull/142214/commits/f76d7039365d98f9f73bee27be549fe2f579c630)

Reply:
ahh, I see, thanks for the explanation!